domain/src/main/java/org/oppia/domain/exploration/ExplorationProgressController.kt

```diff
@@ -408,7 +408,8 @@ class ExplorationProgressController @Inject constructor(
             finishLoadExploration(exploration!!, explorationProgress)
             AsyncResult.success(explorationProgress.stateDeck.getCurrentEphemeralState())
           } catch (e: Exception) {
-            exceptionsController.logException(e, oppiaClock.getCurrentCalendar().timeInMillis)
+            exceptionsController
```

Comment:
I'd expect the line break to happen at the ``(`` first except when chaining multiple method calls together.

Feel free to resolve this comment after addressing it.

Reply:
Done.